src/terrain/clients/search.clj

```diff
@@ -0,0 +1,18 @@
+(ns terrain.clients.search
+  (:use [terrain.util.transformers :only [secured-params]]
```

Comment:
It doesn't look like `secured-params` is used here yet (but we can leave it for now if we're planning on passing the user params to the search service in the future).

Reply:
oh, you're right. I originally used it, but the structuring using `terrain.util/controller` means I didn't need to, but I forgot to remove the import.